pool/cqrs_command/assignment_command.ml

```diff
@@ -227,5 +227,8 @@ end = struct
           ]
   ;;
 
-  let effects = [ `Create, `TargetEntity `Assignment ]
+  let effects =
```

Comment:
Saving a comment for a waiting_list is still denied.

Reply:
@timohuber missing one found and added.